### PR TITLE
P-5625: add event docs

### PR DESCRIPTION
**Pull request by andrewblaney**
- adds `createThreadEvent` docs
- updates `createCustomerDocs` 
- updates `events` page to describe both event types

### CODE REVIEW

File: pages/graphql/events/create-thread-event.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Create a thread event
+
+<Callout type="info" emoji="️✋">
+  A thread event will only be created in the thread ID provided. If you want an event to appear in
+  all threads for a customer please use a [customer event](/graphql/events/create-customer-event).
+</Callout>
+
+To create a thread event you need a thread ID.
+
+You can get this by [creating a thread](/graphql/threads/create) in Plain, from data in webhooks or other API calls you made. If you want to test this, press **⌘ + K** on any thread and then "Copy thread ID" to get an ID you can experiment with.
```
Comment (preslavmihaylov) [inline pages/graphql/events/create-thread-event.mdx:16]: i think the cmd+k to copy thread id doesn't really work yet, maybe we should add a ticket for this, unless someone fixed it after refinement today

Comment (andrewblaney) [reply pages/graphql/events/create-thread-event.mdx:16]: 😄 I opened a branch to add it and it was right there already - I suspect I spelt thread from this morning

**Review by preslavmihaylov — APPROVED** [review]

File: pages/graphql/events/create-customer-event.mdx [hunk 2]
```diff
@@ -6,9 +6,10 @@ import CreateCustomerEventGraphQL from '../../../snippets/graphql/create-custome
 
 # Create a customer event
 
-<Callout type="error" emoji="🚧">
-  This API is being actively developed and is not yet ready for production usage. If you are interested in using events please get in touch with us at [help@plain.com](mailto:help@plain.com)
-
+<Callout type="info" emoji="️✋">
+  A customer event will be created in all threads that belong to the provided customer ID. If you
+  want an event to appear in a specific thread use a[thread
```
Comment (DLeyland) [inline pages/graphql/events/create-customer-event.mdx:11]: nit:
```suggestion
  want an event to appear in a specific thread use a [thread
```

File: pages/graphql/events/_meta.json [hunk 3]
```diff
@@ -1,3 +1,4 @@
 {
-  "create-customer-event": "Create customer event"
+  "create-customer-event": "Create customer event",
```
Comment (DLeyland) [inline pages/graphql/events/_meta.json:2]: nit: Can these page titles either have an 'a' in like 'Create a customer event' or be plural e.g 'Create customer events'

**Review by DLeyland — APPROVED** [review]
Nice 👍

**Pull request merged**